### ISSUE TITLE
Partition to store images on the boot USB

### ISSUE THREAD

**Issue by jlydia** [rescuezilla/rescuezilla#288]
Currently, we're having to use two USB drives, one to boot RescueZilla and another to store Backup Images. Is it possible to be able to have a R/W partition on the boot USB drive, where we can save Backed up Images?

**Comment by shasheene** [comment]
This feature is called ["persistence" (task #8)](https://github.com/rescuezilla/rescuezilla/issues/8). I will have some solutions for available for Rescuezilla v2.3, which at this stage should be released on November 28 2021 (a bit more than a week from now).

That issue has a lot of discussion about some of the approaches, and their drawbacks. Due to limitations in Rescuezilla v2.2 I need to do some further analysis, testing and development before officially supporting the approaches described there.

**Comment by mzky** [comment]
Temporary solution：
UltraISO  has a hidden boot partition feature.This gives the USB two partitions to use after writing
![image](https://user-images.githubusercontent.com/13345233/145317934-7a47fdb6-8741-4948-9730-bb2766c0163b.png)

**Comment by shasheene** [comment]
@mzky that might create a persistent partition (like the solution using Rufus), but I haven't confirmed whether the Rescuezilla frontend software supports that approach particularly well.

Hopefully your suggestion (and the others on #8) works for you and any others. Pretty sure you might need to select "Load USB into RAM" to get things to work!!

**Comment by mzky** [comment]
Yes, I neglected "Load USB into RAM", it solves this problem perfectly.

**Comment by jacobm3** [comment]
"Load USB into RAM" doesn't work for me when I also need to specify "graphical fallback" mode.

What does work for me is installing Ventoy on a flash drive with space at the end for a partition dedicated to images. That results in a partition layout like this:

1. Ventoy data/ISO images partition
2. Ventoy boot partition
3. Rescuezilla images partition 

I copy the Rescuezilla ISO file onto the first Ventoy data partition along with any other boot images I want (shred-os, memtest86, live linux distros, etc.). Then I copy my Rescuezilla backup image directories to the 3rd partition filesystem.

This allows me to skip loading the USB to ram so I can use graphical fallback mode and still restore from images on the same flash device I booted Rescuezilla from.

**Comment by jlydia** [comment]
What tool did you use to create your partitions, and what size for 1 & 2?

**Comment by jacobm3** [comment]
I used the Windows 10 "Create and format hard disk partitions" tool, but you could use anything that can make a partition and an exfat filesystem.

I set the partition sizes based on the size of each flash drive I'm using. Ventoy sets its boot partition size automatically. My windows images usually compress down to 12GB with zstandard-3, so I'll set partition 3 to anywhere from 20GB to 120GB. You specify that as how much space to leave in the Ventoy config.

Ventoy and Rescuezilla are a great combination together.
